doc/examples/tissue_classification.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+"""
+=======================================================
+Tissue Classification of a T1-weighted Strutural Image
+=======================================================
+This example explains how to segment a T1-weighted structural image using a MRF
```

Comment:
This will not render correctly without a line break here (between the title and the following line)